src/transforms/rename.jl

```diff
@@ -0,0 +1,37 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+"""
+  Rename(:col₁ => :newcol₁, :col₂ => :newcol₂, ..., :col₁ => :newcolₙ)
+
+The transform that renames `col₁` to `newcol₁`, `col₂` to `newcol₂`, ...
+"""
+struct Rename <: Stateless
+  names::Dict{Symbol,Symbol}
+end
+
+function apply(transform::Rename, table)
+  oldnames = Tables.columnnames(table)
+  newnames = map(oldnames) do oldname
+    if oldname in keys(transform.names) 
+      transform.names[oldname]
+    else
+      oldname
+    end
```

Comment:
This could be a simple:
```(oldname in keys(transform.names))  ? transform.names[oldname] : oldname```
or even:
`if (oldname in keys(transform.names)) transform.names[oldname] else oldname end`

Reply:
The ternary operator is nice, specially if you save a new variable `names = transform.names` before the map to shorten it up.